packages/astro/src/@types/astro.ts

```diff
@@ -2408,6 +2408,133 @@ export interface AstroUserConfig {
 		 * For a complete overview and the full API reference, see [the Content Layer API RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/0047-content-layer.md) and [share your feedback](https://github.com/withastro/roadmap/pull/982).
 		 */
 		contentLayer?: boolean;
+
+		/**
+		 * @docs
+		 * @name experimental.tsconfig
+		 * @type {object}
+		 * @default `undefined`
+		 * @version 4.15.0
+		 * @description
+		 *
+		 * Enables the generation of `.astro/tsconfig.json`. This allows to exclude the `outDir` by default to avoid `astro check` false positives.
+		 *
+		 * If you enable this option, you'll need to update your `tsconfig.json` with a new `extends` value:
+		 *
+		 * ```json title="tsconfig.json" del={2} ins={3}
+		 * {
+		 *   extends: 'astro/tsconfigs/base',
+		 * 	 extends: ['astro/tsconfigs/base', './.astro/tsconfig.json']
```

Comment:
``````suggestion
		 * ```json title="tsconfig.json" ins="'./.astro/tsconfig.json'"
		 * {
		 * 	 extends: ['astro/tsconfigs/base', './.astro/tsconfig.json']
``````
This looks simpler to me? Just highlight the new entry? Or is it because of the array you wanted an entirely new line?